public/assets/homepage.js

```diff
@@ -203,6 +203,17 @@ $( document ).ready(function() {
         logWebpageActivity("Click_module=StartExploring_location=Index");
     });
 
+    //Triggered when the city or mapathon links are clicked
+    // Logs"Click_module=ExploreMapLink_Location=<cityName or "mapathon">
```

Comment:
This comment is also slightly out of date now (you're logging "mapathonLink" not just "mapathon"). Also missing a space after the word "Logs".